product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api-and-apply-qos/common-utils/src/main/java/org/wso2/am/scenario/test/common/APIRequest.java

```diff
@@ -35,6 +41,23 @@ public APIRequest(String name, String context, String visibility, String version
         constructSwagger();
     }
 
+    public APIRequest(String name, String context, String visibility, String version, String resource, String tiersCollection, URL endpointUrl) {
+        this.name = name;
+        this.context = context;
+        this.visibility = visibility;
+        this.version = version;
+        this.resource = resource;
+        this.tiersCollection = tiersCollection;
+        try{
```

Comment:
Formatting issue.

Reply:
Fix it in dc9a5c9555745294c47d4dc0e6cad63bdf5e4410